tests/cephfs/lib/cephfs_attributes_lib.py

```diff
@@ -0,0 +1,269 @@
+import json
+import os
+import random
+import string
+import unicodedata
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CephFSAttributeUtilities:
+    def __init__(self, ceph_cluster):
+        self.ceph_cluster = ceph_cluster
+        self.mons = ceph_cluster.get_ceph_objects("mon")
+        self.mgrs = ceph_cluster.get_ceph_objects("mgr")
+        self.osds = ceph_cluster.get_ceph_objects("osd")
+        self.mdss = ceph_cluster.get_ceph_objects("mds")
+        self.clients = ceph_cluster.get_ceph_objects("client")
+        self.installer = ceph_cluster.get_nodes(role="installer")[0]
```

Comment:
This variables are part of FsUtils. can we access from self.fs_util object it self